source/output.h

```diff
@@ -510,13 +511,13 @@ int outputHTML2(	double PRE_TIME[NumAlgo][NumPatt],
    				int best = 0;
    				if(TIME[algo][il]==OPTIMAL[il]) best=1;
    				fprintf(fp,"<td><center>");
-   				if(pre)  fprintf(fp,"<div class=\"pre_time\">%.2f</div>",PRE_TIME[algo][il]);
+                fprintf(fp,"<div class=\"pre_time\" style=\"display:%s\">%.2f</div>",(pre? "block" : "none"), PRE_TIME[algo][il]);
```

Comment:
to improve readability I suggest you to save the ternary operation in a separate variable, like
```c++
const string visibility = pre ? "block" : "none";
```

Reply:
value is cached in a const char * for pre and dif.